examples/xonotic/main.go

```diff
@@ -18,22 +18,93 @@ import (
 	"log"
 	"time"
 
```

Comment:
The following imports can be moved up one line to be next to log and time.

Reply:
Done.